Overhaul Enable-BitLocker doc page (#2866)
MicrosoftDocs/windows-powershell-docs: docset/winserver2016-ps/bitlocker/Enable-BitLocker.md

- **TPM and startup key:** BitLocker uses a combination of the TPM and a USB flash drive that
  contains the external key.

- **Startup key:** BitLocker uses a USB flash drive that contains the external key.

- **Password:** BitLocker uses a password.

- **Recovery key:** BitLocker uses a recovery key stored as a specified file.

- **Recovery password:** BitLocker uses a recovery password.

- **Active Directory Domain Services (AD DS) account:** BitLocker uses domain authentication.

You can specify only one of these methods or combinations when you enable encryption, but you can
use the **Add-BitLockerKeyProtector** cmdlet to add other protectors.

For a password or PIN key protector, specify a secure string. You can use the
**ConvertTo-SecureString** cmdlet to create a secure string. You can use secure strings in a script
and still maintain confidentiality of passwords.

We strongly recommend specifying the encryption method. By default, BitLocker uses XTS-AES-128. You
can opt XTS-AES-256 for stronger security. However, if you are encrypting a removable media and
intend to use it on Windows 8.1 or Windows Server 2012 R2, you must opt either AES-128 or AES-256

> for backward compatibility. You may request hardware encryption but Microsoft strongly advices
> against it. For further guidance, see the "[ADV180028 Security Advisory][1]".

Reviewer: ```suggestion
for backward compatibility. You may request hardware encryption but we strongly advise
against it. For further guidance, see the [ADV180028 Security Advisory][1].
```

This cmdlet returns a BitLocker volume object. If you choose recovery password as your key protector
but do not specify a 48-digit recovery password, this cmdlet generates a random one for you, and
stores it in the **RecoveryPassword** field of the **KeyProtector** attribute of the BitLocker
volume object.

If you use startup key or recovery key as part of your key protector, provide a path to store the
key. This cmdlet stores the name of the file that contains the key in the **KeyFileName** field of
the **KeyProtector** field in the BitLocker volume object.

> If you use the **Enable-BitLocker** cmdlet on an encrypted volume or on a volume that with

Reviewer: ```suggestion
If you use the **Enable-BitLocker** cmdlet on an encrypted volume or on a volume with
```

encryption in process, it takes no action. If you use the cmdlet on a drive that has encryption
paused, it resumes encryption on the volume.

By default, this cmdlet encrypts the entire drive. If you use the *UsedSpaceOnly* parameter, it only

> encrypts the used space on the disk. This option can significant reduce encryption time.

Reviewer: ```suggestion
encrypts the used space on the disk. This option can significantly reduce encryption time.
```

It is common practice to add a recovery password for an operating system volume using the
**Add-BitLockerKeyProtector** cmdlet, save the recovery password using the
**Backup-BitLockerKeyProtector** cmdlet, and then enable BitLocker on that volume. This procedure
ensures that you have a recovery option.

> For an overview of BitLocker, see "[BitLocker Drive Encryption Overview][2]" on Microsoft Docs.

Reviewer: ```suggestion
For an overview of BitLocker, see the [BitLocker Drive Encryption Overview][2]".
```
